### PR TITLE
support migrate python functions

### PR DESCRIPTION
**Pull request by ondrajodas**
Jira - https://keboola.atlassian.net/browse/CM-702

### CODE REVIEW

File: src/Migrate.php [hunk 1]
```diff
@@ -1601,6 +1607,40 @@ private function buildFunctionQuery(array $function, array $functionParams): str
...
+RUNTIME_VERSION=%s
+HANDLER=%s
+%s
+%s
+AS 
+$$
+%s
+$$
+;
+SQL;
```
Comment (pivnicek) [inline src/Migrate.php:1625]: 🤯 😄

File: src/Migrate.php [hunk 2]
```diff
@@ -1601,6 +1607,40 @@ private function buildFunctionQuery(array $function, array $functionParams): str
...
+            $functionParams['returns'],
+            $functionParams['runtime_version'],
+            QueryBuilder::quote($functionParams['handler']),
+            $functionParams['imports'] !== '[]' ?
+                'IMPORTS = (' . substr($functionParams['imports'], 1, -1) . ')' :
+                '',
+            $functionParams['packages'] !== '[]' ?
+                'PACKAGES = (' . substr($functionParams['packages'], 1, -1) . ')' :
+                '',
+            trim($functionParams['body'])
```
Comment (pivnicek) [inline src/Migrate.php:1640]: makes sense

**Review by pivnicek — APPROVED** [review]
alright, LGTM, have you tried it?

**Comment by ondrajodas** [comment]
@pivnicek sure, I tried create these three functions:
```SQL
create or replace function addone(i int)
returns int
language python
runtime_version = '3.8'
handler = 'addone_py'
as
$$
def addone_py(i):
  return i+1
$$;

create or replace function multiple_import_files(s string)
returns string
language python
runtime_version=3.8
handler='compute'
as
$$
def compute(s):
  return s
$$;

CREATE OR REPLACE FUNCTION py_udf()
RETURNS VARIANT
LANGUAGE PYTHON
RUNTIME_VERSION = 3.8
PACKAGES = ('numpy','pandas','xgboost==1.5.0')
HANDLER = 'udf'
AS $$
import numpy as np
import pandas as pd
import xgboost as xgb
def udf():
  return [np.__version__, pd.__version__, xgb.__version__]
$$;
```

**Comment by pivnicek** [comment]
pekny

**Pull request merged**